source/playbooks/NIST80053/ssmdocs/NIST80053_AutoScaling.1.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Are these copy-pasted from SC? Please review how CIS 1.4.0 reuses the same code.